tls/assets/configuration/spec.yaml

```diff
@@ -133,5 +133,15 @@ files:
       description: Unique identifier for this instance that is added as a tag to all data emitted.
       value:
         type: string
+    - name: start_tls
+      description: |
+        Send protocol-specific message(s) to switch to TLS for communication.
+        StartTLS is implemented on these protocols: smtp, pop3, imap, ftp, xmpp, xmpp-server,
+        irc, postgres, mysql, lmtp, nntp, sieve, and ldap.
+
+        Currently this checks supports only the below protocols:
+          postgres
+      value:
+        type: string
```

Comment:
You can specify this field is an enum, example [here](https://github.com/DataDog/integrations-core/blob/8571f774c13c86bb127e4aa1f56057e19eba2619/ibm_db2/assets/configuration/spec.yaml#L43-L52) so the check will automatically validate the value